kong/api/endpoints.lua

```diff
@@ -222,7 +222,10 @@ local function query_entity(context, self, db, schema, method)
       return dao[context](dao, size, args.offset, opts)
     end
 
-    return dao[method](dao, self.params[schema.name], size, args.offset, opts)
+    if schema.name == "tags" then
```

Comment:
```suggestion
    local key = self.params[schema.name]
    if schema.name == "tags" then
      key = unescape_uri(key)
    end
```

This will fix https://github.com/Kong/kong/runs/7738245819?check_suite_focus=true